src/simplewallet/simplewallet.cpp

```diff
@@ -2650,13 +2650,20 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
   {
     fail_msg_writer() << tr("failed to get random outputs to mix: ") << e.what();
   }
-  catch (const tools::error::not_enough_money& e)
+  catch (const tools::error::not_enough_unlocked_money& e)
   {
     LOG_PRINT_L0(boost::format("not enough money to transfer, available only %s, sent amount %s") %
```

Comment:
Worth changing this to "not enough unlocked money to transfer, ..." and the same elsewhere?

Reply:
The user facing message is just below with `fail_msg_writer()`, the line you mention is only written to the log file.

For the line written to the log file, there is a difference between "not enough money to transfer, available only..." and "not enough money to transfer, overall balance only...".

So I think it's fine.